facebook-commerce.php

```diff
@@ -938,47 +933,45 @@ public function on_product_trash( $post_id ) {
 	/**
 	 * Deletes a product from Facebook.
 	 *
-	 * @param int $wp_id product ID
+	 * @param int $product_id product ID
 	 */
-	public function on_product_delete( $wp_id ) {
+	public function on_product_delete( $product_id ) {
 
-		$woo_product = new WC_Facebook_Product( $wp_id );
+		$woo_product = new \WC_Facebook_Product( $product_id );
```

Comment:
can we drop instantiating this object? doesn't seem to help us in any way in the context of this method and could be replaced by the result of the more standard `wc_get_product()`

Reply:
similar thing in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1315

Reply:
Yeah, I think that makes sense. We pass that to `get_product_fbid()` but is optional. I'll quickly change that before I merge.

Reply:
I made the updates in 6fc8e49 and b7f4660.